lib/libutee/include/pta_scmi_client.h

```diff
@@ -0,0 +1,72 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2019-2021, Linaro Limited
+ */
+#ifndef PTA_SCMI_CLIENT_H
+#define PTA_SCMI_CLIENT_H
+
+#define PTA_SCMI_UUID {\
+		0xa8cfe406, 0xd4f5, 0x4a2e, \
+		{ 0x9f, 0x8d, 0xa2, 0x5d, 0xc7, 0x54, 0xc0, 0x99 } \
+	}
+
+#define PTA_SCMI_NAME "PTA-SCMI"
+
+enum optee_smci_pta_cmd {
+	/*
+	 * PTA_SCMI_CMD_CAPABILITIES - Get channel capabilities
```

Comment:
What is this channel mentioned several times? Same goes for the channel handle, what's the difference?

Reply:
Looking back why I used channel, i think it's related to Linux scmi driver implementation. There, Linux creates a channel for each 'ringbell'+'shared memory' couple, making a communication channel between an scmi agent (Linux) and an scmi server. An agent can implement several channels, using a shared memory per channel. Linux only implements SMT channels where 
 messages must be posted/processed one after the other, the mailbox cannot multiplex messages.

The word channel in used in scmi Linux driver. Here, for example, in `struct scmi_chan_info` 
https://elixir.bootlin.com/linux/v5.9/source/drivers/firmware/arm_scmi/common.h#L161
or the `::chan_*` fields of `struct scmi_transport_ops` few lines below.
So OP-TEE SCMI PTA allows several _channels_.

As for channel handles, we expect to use [SCP-firmware](https://github.com/ARM-software/SCP-firmware) implementation to build an SCMI server library OP-TEE (or other envs/OSes) could host. In SCP-firmware, _channels_ are assigned a handle at run time.  Client shall get and use this handle. Therefor this _channel_ - to - _channel handle_ conversion API.
With scmi-msg drivers, channel handles are static, can be known from binary (in DT, for linux or u-boot), no real need for such conversion.

Reply:
Thanks this helps. Would you mind adding something along this in the AP?